test/distrib/gcf/python/requirements.txt.base

```diff
@@ -0,0 +1 @@
+google-cloud-pubsub
```

Comment:
Should we also pin this package?

Reply:
This was actually intentional. If a new change to the client libraries breaks in GCF with the newest version of gRPC, we're likely to care about it.

Reply:
Should we consider something like `>=0.0.dev0`?

Reply:
Looks like they're not doing prereleases, but this should pick them up if they ever start. Done.